.github/workflows/check-pak.yaml

```diff
@@ -0,0 +1,67 @@
+# Workflow derived from https://github.com/r-lib/actions/tree/master/examples
+# Need help debugging build failures? Start at https://github.com/r-lib/actions#where-to-find-help
+#
+# NOTE: This workflow is overkill for most R packages and
+# check-standard.yaml is likely a better choice.
+# usethis::use_github_action("check-standard") will install it.
+on:
+  push:
+    branches: [main, develop]
+  pull_request:
+    branches: [main, develop]
+
+name: R-CMD-check
+
+jobs:
+  R-CMD-check:
+    runs-on: ${{ matrix.config.os }}
+
+    name: ${{ matrix.config.os }} (${{ matrix.config.r }})
+
+    strategy:
+      fail-fast: false
+      matrix:
+        config:
+          - {os: macOS-latest,   r: 'release'}
+
+          - {os: windows-latest, r: 'release'}
+          # Use 3.6 to trigger usage of RTools35
+          - {os: windows-latest, r: '3.6'}
+
+          # Use older ubuntu to maximise backward compatibility
+          - {os: ubuntu-18.04,   r: 'devel', http-user-agent: 'release'}
+          - {os: ubuntu-18.04,   r: 'release'}
+          - {os: ubuntu-18.04,   r: 'oldrel-1'}
```

Comment:
"release" is the latest released r-version (like 4.1.2 now), "devel" is the next development version (e.g. 4.1.3 or 4.2.0)... and what is "oldrel-1"? 4.1.1 or 4.0.x or 3.6.5?

Reply:
devel is `4.2.0`

Rest are these versions:

``` r
library(rversions)

r_oldrel()
#>     version                date        nickname
#> 122   4.0.5 2021-03-31 07:05:15 Shake and Throw

r_release()
#>     version                date    nickname
#> 125   4.1.2 2021-11-01 08:05:12 Bird Hippie
```

<sup>Created on 2021-11-09 by the [reprex package](https://reprex.tidyverse.org) (v2.0.1)</sup>